src/main/java/dev/ebrydeu/spring_boot_library/repositrories/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+package dev.ebrydeu.spring_boot_library.repositrories;
+
+import dev.ebrydeu.spring_boot_library.domain.entities.User;
+import org.springframework.data.repository.ListCrudRepository;
+import java.util.List;
+
+public interface UserRepository extends ListCrudRepository<User, Long> {
+    List<User> findAll();
+    List<User> findUserById(Long id);
```

Comment:
Remove it. `CrudRepository` interface already have this methods no point to override.

Reply:
Correctly pointed. Deleted